configs/prometheus/prometheus.yaml

```diff
@@ -101,3 +101,20 @@ scrape_configs:
       - source_labels: [__meta_kubernetes_pod_container_port_number]
         action: keep
         regex: 9\d{3}
+  - job_name: 'kubernetes-cadvisor'
+    scheme: https
+    tls_config:
+      ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+    kubernetes_sd_configs:
+      - role: node
+    relabel_configs:
+      - action: labelmap
```

Comment:
[This comment](https://github.com/giantswarm/kubernetes-prometheus/pull/73/files#r142479485) may apply here as well